src/applications/vaos/sass/vaos.scss

```diff
@@ -86,64 +86,61 @@
     margin: 8px;
   }
 
-  button {
-    width: 100%;
-    color: $color-base;
-    background-color: $color-gray-lightest;
-    border: 1px solid $color-gray-medium;
-    border-radius: 5px;
-    padding: 12px;
-    margin: 0;
-
-    &:hover {
-      border-color: $color-primary-darker;
+  &.vaos-calendar__cell-current {
+    margin-bottom: 12px;
+    @include media($small-screen) {
+      margin-bottom: 8px;
     }
   }
 
-  &.vaos-calendar__cell-selected button {
+  &.vaos-calendar__cell-current > button,
```

Comment:
Ideally these aren't underneath `vaos-calendar__calendar-day`. There might be some further tweaking to make that work, but we can look at that later if it's not easy to change.

Reply:
removed `vaos-calendar__day--current > button` and `vaos-calendar__day--selected > button` from underneath `vaos-calendar__calendar-day` and placed them at the root level